zorg/jenkins/jobs/jobs/lldb-cmake-matrix

```diff
@@ -21,7 +21,7 @@ pipeline {
         }
     }
     environment {
-        MATRIX_COMMON_BUILD_PARAMETERS = '--cmake-flag=-DLLVM_TARGETS_TO_BUILD=X86'
+        MATRIX_COMMON_BUILD_PARAMETERS = '--cmake-flag=-DLLVM_TARGETS_TO_BUILD=AArch64'
```

Comment:
Hmm I think we want this to be `X86;AArch64`

Only specifying one of them means we wouldn't run some of the x86 tests on ARM machine and vice-versa. But we do want that test-coverage. They don't actually require an x86 host